src/InfoBoxes/InfoBoxWindow.cpp

```diff
@@ -442,6 +447,8 @@ InfoBoxWindow::OnMouseDouble(PixelPoint p)
 bool
 InfoBoxWindow::OnMouseMove(PixelPoint p, unsigned keys)
 {
+  SetFocus();
```

Comment:
Uh-oh, forcing keyboard focus on mouse move? That sounds like a very bad idea, and will confuse lots of users.

Reply:
The idea here is, with stefly you have a very small courser and to move this during flight is a little bit tricky. The hover would show the user directly that the courser is over the infobox. But I understand that this is a little bit confusing. 
Would it be better to have it configurable or ifdef openvario?

Reply:
There is no compile-time switch for OpenVario.
Does the OpenVario even have a mouse? It has a touch screen, doesn't it?
`OnMouseMove()` triggers on touch screens only while the finger is down. Why would somebody want the keyboard focus to follow the finger? Tapping the finger, ok, but while moving the finger?
None of this makes any sense to me, and your text doesn't explain any of it.

Reply:
The OpenVario doesn't have a touchscreen, but instead is often used with a remote control stick emulating a mouse (four arrow keys move the cursor + mouse click).
I think that this feature is useful only on non-touchscreen devices with such a cursor, so it should be guarded by a compile-time switch or configuration option?

Reply:
Why would that be useful? If you want keyboard focus, click.

Reply:
Oh, and emulating a pointing device with digital arrow keys sounds like the most horrible UI desaster I could imagine in a cockpit. This just doesn't make sense; it takes away too much of precious attention from the pilot, who should be watching outside instead of watching the mouse cursor reach a certain position on the screen.

Reply:
Some kind of highlighting if the mouse hovers over a clickable object is common practice and helps signalizing that clicking is a possible action thus I would consider it useful. If this should be done by SetFocus I can't answer (as I don't know the implications yet).
Or maybe I didn't understand the patch.

Reply:
Agree with highlighting, but that's not what this code submission is about. It is about changing keyboard focus, which I consider a bad idea.

Reply:
I agree, thanks for the explanation (you already explained it in your first message but I didn't understand).

Reply:
This was mentioned in the posts, it is possible with the steFly remote to move a mouse courser with the cross, which emulates the courser keys. My idea was in turbolent condition to give visual feedback, when the info box is reached with the mouse courser.
But I understand that this is not in the usability of XCSoar, and I will remove it in my next code clean up.
In addition with the "infoboxfocus" it is possible to set a key that focus to the last selected infobox is set.